vertical-pod-autoscaler/enhancements/4016-in-place-updates-support/README.md

```diff
@@ -254,6 +257,49 @@ tested in the following scenarios:
 * In-place update will fail but `CanEvict` is false, pod should not be evicted.
 * In-place update will fail but `EvictionRequirements` are false, pod should not be evicted.
 
+### Upgrade / Downgrade Strategy
+
+#### Upgrade
+
+On upgrade of the VPA to 1.4.0 (tentative release version), nothing will change,
+VPAs will continue to work as before.
+
+Users can use the new `InPlaceOrRecreate` by enabling the alpha Feature Gate (which defaults to disabled)
+by passing `--feature-gates=InPlaceResize=true` to the updater and admission-controller components and setting
+their VPA UpdateMode to use `InPlaceOrRecreate`.
+
+#### Downgrade
+
+On downgrade of VPA from 1.4.0 (tentative release version), nothing will change.
+VPAs will continue to work as previously, unless, the user had enabled the feature
+gate. In which case downgrade could break their VPA that uses `InPlaceOrRecreate`.
+
+### Feature Enablement and Rollback
+
+#### How can this feature be enabled / disabled in a live cluster?
+
+- Feature gate name: InPlaceResize
+- Components depending on the feature gate:
+  - admission-controller
+  - updater
+
+Disabling of feature gate `InPlaceResize` will cause the following to happen:
```

Comment:
Actually, one other thing that just came to me, we could call it `InPlaceOrRecreate` here and it would be directly tied to what we are adding to the API.

I'm OK either way though.

Reply:
🤦 it feels like the name was staring us in the face the whole time. Good suggestion! I've made the change